app/translations.js

```diff
@@ -2295,6 +2298,9 @@ const translations = {
     'fetch-new-route': 'Hae uusi reitti',
     'few-seats-available': 'Joitakin istumapaikkoja vapaana',
     finnish: 'Suomi',
+    follow: 'Matkan seuranta',
+    followtheitinerary: 'Matkalle',
+    followyourouteonscreen: 'Löydä perille ohjatusti!',
```

Comment:
Use more consistent id:s. Route means P, IC455, bus 500 etc. For example:
- navigation-header
- start-navigation
- navigation-description

Reply:
Let's use those ones.